test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from bouncer import Bouncer, SecretManagerException
+
+
+class TestBouncer(unittest.TestCase):
+
+    def setUp(self):
+        pass
```

Comment:
I think in setUp should create a new secret (with unique secret name) in Secrets Manager for the purpose of the test and remove it in tearDown when the test is finished.
This test should not depend on the presence or contents of "commons/dev/whitelist"